vcdh5cc.l10n

```diff
@@ -0,0 +1,27 @@
+#
+# This file lists all files that should be localized for a release.
+# The file syntax and the 'l10n-util' command are described on the Wiki page
+#
+#    https://wiki.eng.vmware.com/Build/L10NSupport
+#
+# When updating this file, its contents should be verified prior to
```

Comment:
I'm not sure that this process is what you'll want to follow in OSS??

Reply:
Sarah Quao, from the localization team is asking their team if they need to come up with a new standard. For now, they said they don't have a process but did agree that none of these comments should stay here since they are talking about internal VMware scripts and links.